basic/Area_of_triangle.py

```diff
@@ -0,0 +1,13 @@
+# Python program to calculate area of triangle
+
+# Take base and height inputs from the user
+
+base = int(input("Enter lenght of the base : "))
```

Comment:
from next time, read input from main block and solve the problem using a function call